sbar.c

```diff
@@ -53,6 +53,7 @@ cvar_t  sbar_drawsigils     = {"scr_sbar_drawsigils", "1"};
 cvar_t  sbar_drawhealth     = {"scr_sbar_drawhealth", "1"};;
 cvar_t  sbar_drawarmor      = {"scr_sbar_drawarmor",  "1"};;
 cvar_t  sbar_drawammo       = {"scr_sbar_drawammo",   "1"};;
+cvar_t  sbar_lowammo       = {"scr_sbar_lowammo",   "5"};;
```

Comment:
Some small notes, please correct nbr of spaces for alignment (not tabs if used, on phone so only see its not aligned) :)

Reply:
Oh and feel free to remove the extra unnecessary semicolons on those rows